vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -310,3 +310,45 @@ def diverging_stacked_bar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
         fig.add_hline(y=0, line_width=2, line_color="grey")
 
     return fig
+
+
+@capture("graph")
+def lollipop(data_frame: pd.DataFrame, **kwargs):
+    """Creates a lollipop based on px.scatter.
+
+    A lollipop chart is a variation of a bar chart where each data point is represented by a line and a dot at the end
+    to mark the value.
+
+    Inspired by: https://community.plotly.com/t/how-to-make-dumbbell-plots-in-plotly-python/47762
+
+    Args:
+        data_frame: DataFrame for the chart. Can be long form or wide form.
+            See https://plotly.com/python/wide-form/.
+        **kwargs: Keyword arguments to pass into px.scatter (e.g. x, y, labels).
+            See https://plotly.com/python-api-reference/generated/plotly.scatter.html.
+
+    Returns:
+        go.Figure: Lollipop chart.
+    """
+    # Unlike the column_and_line chart, where all traces hold equal significance, here the traces differ in importance.
+    # The primary scatter plot is the main trace, while the additional traces merely serve as connecting lines.
+    # Therefore, should we apply the kwargs solely to the main scatter plot, as illustrated below?
+    fig = px.scatter(data_frame, **kwargs)
```

Comment:
One question: In this case, there is one main scatter trace and several subsidiary scatter traces that only add lines. I assume in this case, we only add `kwargs` to the main trace and we don't enable the the `kwargs` for the subsidiary scatter traces?

Reply:
>  I assume in this case, we only add kwargs to the main trace and we don't enable the the kwargs for the subsidiary scatter traces?

I think this is totally correct 👍.